### ISSUE TITLE
Os trigonum

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11189]
**Preferred term label:**
Os trigonum

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1457873
Lateral ankle x-ray with foot in plantarflexion revealing accessory ossicle at the posterior aspect of the talus, between the posterior distal tibia and calcaneal tuberosity

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Os trigonum syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001349
- label: Os trigonum
- parents: [Abnormal talus morphology (HP:0008365)](https://hpo.jax.org/app/browse/term/HP:0008365)
- pmid: PMID:39586987
- def: Os trigonum (OT) is one of the most common accessory ossicles in the foot and ankle region. In clinical practice, OT is a considerable factor that may predispose patients to posterior ankle impingement syndrome (PAIS). Lateral ankle x-ray with foot in plantarflexion reveals an accessory ossicle at the posterior aspect of the talus, between the posterior distal tibia and calcaneal tuberosity.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11189
- synonyms:
